src/App.tsx

```diff
@@ -1,28 +1,34 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import classes from 'App.module.scss';
-import { Route, Routes, useLocation } from 'react-router-dom';
+import { Route, Routes } from 'react-router-dom';
 import Login from 'pages/Login/Login';
-import BooksOverview from 'pages/BooksOverview/BooksOverview';
-import SingleBook from 'pages/SingleBook/SingleBook';
 import Navbar from 'components/Navbar/Navbar';
-import Dashboard from 'pages/Admin/Dashboard/Dashboard';
 import Profile from 'pages/Profile/Profile';
 import { RoleContext } from 'contexts/roleContext';
 import UsersList from 'components/UsersList/UsersList';
+import RentalsOverview from 'components/RentalsOverview/RentalsOverview';
+import DashboardLayout from 'layouts/DashboardLayout';
+import Book from 'pages/Book/Book';
+import BooksOverview from 'pages/BooksOverview/BooksOverview';
 
 const App = () => {
-  const { pathname } = useLocation();
+  const { role } = useContext(RoleContext);
 
   return (
     <div className={classes['c-app']}>
-      {pathname !== '/' && <RoleContext.Consumer>{({ role }) => <Navbar />}</RoleContext.Consumer>}
+      {role === 'USER' && <Navbar />}
```

Comment:
I think i suggested here to use isRoleUser() method to check this since you created it in service it should be reused